components/__tests__/UnitInput.test.ts

```diff
@@ -0,0 +1,163 @@
+import { mount } from '@vue/test-utils';
+// @ts-ignore
```

Comment:
Were these added due to the removal of the `module-resolver` in the babel config?